SmartDeviceLinkTests/RPCSpecs/NotificationSpecs/SDLOnSystemRequestSpec.m

```diff
@@ -74,6 +75,45 @@
         expect(testNotification.offset).to(beNil());
         expect(testNotification.length).to(beNil());
     });
+
+    it(@"Should set and get url correctly (zero length url)", ^ {
+        SDLOnSystemRequest* testRequest = [[SDLOnSystemRequest alloc] init];
+        NSString *testURL = @"";
+        testRequest.url = testURL;
+        expect(testRequest.url).to(equal(testURL));
+    });
+
+    it(@"Should set and get url correctly (1000 sym long url)", ^ {
+        const int size = 1000;
+        SDLOnSystemRequest* testRequest = [[SDLOnSystemRequest alloc] init];
+        NSMutableString *testURL = [NSMutableString stringWithCapacity:1024];
+        [testURL setString:SDLTestURL];
+        while(size > testURL.length) {
+            [testURL appendString:@"/testcomponent"];
+        }
+        if (size < testURL.length) {
+            const NSRange range = NSMakeRange(size, testURL.length - size);
+            [testURL deleteCharactersInRange:range];
+        }
+        testRequest.url = testURL;
+        expect(testRequest.url).to(equal(testURL));
```

Comment:
Also test what happens when turning this string into an `NSURL` (it should create one)